src/transformers/models/t5/modeling_flax_t5.py

```diff
@@ -1280,7 +1330,7 @@ def __call__(
         encoder_outputs=None,
         output_attentions=None,
         output_hidden_states=None,
-        return_dict=None,
+        return_dict: bool = True,
```

Comment:
Once the aforementioned `return_dict` changes are made to the `FlaxT5BlockCollection` this change won't be necessary :)